setup.cfg

```diff
@@ -10,7 +10,7 @@ maintainer = Robert A. McLeod
 maintainer_email = robbmcleod@gmail.com
 url = https://github.com/pydata/numexpr
 long_description = file: README.md
```

Comment:
This line should be `long_description = file: README.rst`.

Reply:
Thanks @stephan-cr